wcivf/apps/feedback/templates/feedback/feedback_form.html

```diff
@@ -13,6 +13,14 @@ <h2>
       {{ choice.tag }}
       <label class="link-button" for="{{ choice.id_for_label }}">{{ choice.choice_label }}</label>
     {% endfor %}
+    
+    <div class="comments_container">
+      <p>
+        What other websites/sources have you used to 
+        research the election?
+      </p> 
+        {{ feedback_form.comments }}
```

Comment:
I think this should be `feedback_form.sources`

Reply:
That's what I assumed too, but the text box didn't appear. And that's because I had it hidden as you mention below. Thanks!